servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -910,6 +910,12 @@ void RenderForwardMobile::_render_scene(RenderDataRD *p_render_data, const Color
 		global_pipeline_data_required.use_lightmaps = true;
 	}
 
+	if (global_surface_data.screen_texture_used || global_surface_data.depth_texture_used) {
+		// Just called to create the framebuffer since we know we will need it later.
+		rb_data->get_color_fbs(RenderBufferDataForwardMobile::FB_CONFIG_RENDER_PASS);
```

Comment:
Doesn't `rb_data` need to be checked to be valid?

Reply:
Good catch!